tensorflow/python/feature_column/feature_column_v2.py

```diff
@@ -537,8 +537,8 @@ def __init__(self,
 
         with `sparse_combiner` as "mean", the linear model outputs conceptly are
         ```
-        y_0 = 1.0 / 2.0 * ( w_a + w_ b) + w_c + b_0
-        y_1 = w_d + 1.0 / 3.0 * ( w_e + w_ f + w_g) + b_1
+        y_0 = 1.0 / 3.0 * ( w_a + w_ b + w_c) + b_0
```

Comment:
I think the formula is correct. We take the mean for the 0th row in the first case and the 1st row in the second case.